sdk/@launchdarkly/observability-node/src/client/ObservabilityClient.ts

```diff
@@ -32,7 +32,10 @@ import {
 	registerInstrumentations,
 } from '@opentelemetry/instrumentation'
 import { CompressionAlgorithm } from '@opentelemetry/otlp-exporter-base'
-import { processDetectorSync, Resource } from '@opentelemetry/resources'
+import {
+	processDetector,
+	resourceFromAttributes,
```

Comment:
The `Resource` type is now an interface instead of a class. So you have to use this factory function instead.